src/main/play-doc/intro/Clustering-Considerations.md

```diff
@@ -0,0 +1,42 @@
+# Typesafe ConductR %PLAY_VERSION%
+
+## Cluster Resiliency Considerations
+
+In order to achieve cluster resilience when using ConductR, it’s recommended for the cluster to be separated into at least 3 distinct parts, e.g.
+
+* If you are using Amazon, this would be spreading the number of nodes evenly in 3 different availability zones
+* If you are hosting the machines yourself, spread the number of nodes evenly in 3 different networks if it’s possible to do so
+* Or if you only have 1 network, try to split the nodes among 3 different physical machine
+
+Separating the number of nodes evenly into 3 distinct parts will allow best chance of cluster recovery when some nodes of the cluster become unreachable.
+
+## Cluster Split (Split Brain) Scenario Recovery
+
+ConductR utilizes Akka SBR (Split Brain Recovery) which is made available as part of [Akka Typesafe RP](http://www.typesafe.com/products/typesafe-reactive-platform) version `15v01p05`.
+
+The Akka SBR strategy adopted by ConductR is keep-majority.
+
+The `keep-majority` strategy ensures when cluster split occurs:
+
+* The partition which has the majority of the nodes are kept running
+* Other partition(s) which has less than majority of the nodes will have each of their nodes terminated gracefully.
+
+## Cluster Split Automated Recovery
+
+In the RPM version of the ConductR, `systemd` is utilised to ensure the ConductR process keeps running. Akka SBR and `systemd` in conjunction will allow ConductR to perform automated recovery should the event of cluster split occur.
+
```

Comment:
Making sure 'Restart' is configured in ConductR RPM's systemd is out of the outstanding task still needs to be done